src/main/resources/openapi.yaml

```diff
@@ -0,0 +1,335 @@
+openapi: 3.0.0
+info:
+  title: Passport Office API
+  description: Passport Office API providing passport and people managment
+  version: 0.0.1
+servers:
+  - url: localhost:8080/v1
+paths:
+  /persons:
+    get:
+      tags:
+        - person
+      summary: Returns a list of persons
+      description: People managment
+      operationId: getPersons
+      responses:
+        '200':
+          $ref: '#/components/requestBodies/PersonsBody'
+    post:
+      tags:
+        - person
+      summary: Creates a person
+      description: People managment
+      operationId: createPerson
+      requestBody:
+        $ref: '#/components/requestBodies/PersonBody'
+      responses:
+        '201':
+          description: A created person
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/requestBodies/PersonBody'
+  '/persons/{personId}':
+    get:
+      tags:
+        - person
+      parameters:
+        - name: personId
+          in: path
+          description: ID of the person
+          required: true
+          schema:
+            type: integer
+      summary: Returns a person
+      description: People managment
+      operationId: getPerson
+      responses:
+        '200':
+          $ref: '#/components/requestBodies/PersonBody'
+    put:
+      tags:
+        - person
+      parameters:
+        - name: personId
+          in: path
+          description: ID of the person
+          required: true
+          schema:
+            type: integer
+      summary: Updates a person
+      description: People managment
+      operationId: putPerson
+      requestBody:
+        $ref: '#/components/requestBodies/PersonBody'
+      responses:
+        '200':
```

Comment:
There might be `400` status in case of request is malformed
There also might be `409` in case of unique constraint violation

Reply:
The responses have been added